nx/blocks/importer/index.js

```diff
@@ -13,6 +13,8 @@ const LINK_SELECTORS = [
   'a[href*=".svg"]',
   'img[alt*=".mp4"]',
 ];
+// For any case where we need to find SVGs outside of any elements // in their text.
+const LINK_SELECTOR_REGEX = /https:\/\/[^"'\s]+\.svg/g;
```

Comment:
I wonder if we should preemptively also fetch other assets? I only seen this case for SVGs

Reply:
I'd be fine with that. It's pretty rare, though... borderline non-existent.